### PR TITLE
Ports removes all budget cards except for cargo card, gives cargo small income

### PR DESCRIPTION
**Pull request by VictorVow**
https://github.com/tgstation/tgstation/pull/47256

## **The What**

Removes every departmental budget card from the game, **INCLUDING** the Cargo Departmental Card. Most departmental budgets (besides Cargo) are now only used for paychecks to employees.

Cargo Budget now receives an EXTREMELY small (read: 500 credits) paycheck per cycle.
## **The How**

Removed the individual budget cards items in 'card_ids.dm', changed the secure closet files to make it so said cards (that no longer exist) spawn in their corresponding head's locker. Changed the 'economy.dm' file to have Cargo get the 500 credit paycheck.
## **The Why**

_Oh, boy._

Firstly, the department budgets are super dumb and easy to abuse. Secondly, they were never actually used by their departments except for when the QM would stop by an ask for the card -- which he then got literal tens of thousands of dollars every paycheck to piss around with. So, department budgets (again, with the exception of the Cargo budget) can now ONLY be used to dish out funds for employee's paychecks. No more, no less.

What this means is that Cargo now has to do exports and bounties for any meaningful amount of credits -- if things are going slow, they do have a very small amount of reliable income to work with (enough to buy a medkit every few minutes, if medbay is useless. But not enough to snag combat shotties unless they wait for an extremely long period of time - 80 minutes - and assuming that they have no Cargonians taking away from the budget for paychecks.) Basically, to be useful, they HAVE to do exports and bounties.

So, just to reiterate -- if I want Cargo to actually do exports and bounties, why do they get 500 credits every paycheck? Firstly, this 500 credit paycheck is usually almost completely negated by cargo employee's paycheck. Especially if the department is large. The remaining funds are a laughable amount, meaning Cargo has to save for a _long_ time if they have no source of income outside of their paycheck. To get any meaningful amount of money, they need to start exporting. They can't rely on mooching off of other departmental cards.

'Hold on a minute', you may be asking, 'You didn't actually remove the department's budgets, but just the card. Are you retarded or something?' While the answer may be yes, the retainment of the department budgets is intentional for a simple reason; without them, nobody would get paychecks. However, these 'budgets' now no longer serve ANY purpose besides funding paychecks. Meaning departments can pool together their paychecks to buy things for Cargo, but the money isn't enough to let Cargo get SWAT crates 10 minutes after roundstart.

**TL;DR:** Cargo is more inclined to do exports and bounties since they can't mooch off of other department budgets and have no steady source of income outside of said exports and bounties.
## Changelog

:cl: genessee596
del: Removed all departmental budget cards including Cargo card
tweak: Cargo now gets a flat 500 credits per paycheck, not including exports or bounties
/:cl:

### CODE REVIEW

**Comment by jlsnow301** [comment]
Will not miss department cards nor the hassling for them

**Comment by MarkSuckerberg** [comment]
No, please no. Budget cards are needed for proper operation of competent department heads. They need them to order from cargo, buy expensive things such as advanced medical kits, and make being payed make slightly more sense than the way you suggest. I veto or downvote this PR as much as possibly can be done.

**Comment by MarkSuckerberg** [comment]
Also you're using /tg/'s pr format, not bee's

**Comment by HeartwawyHorror** [comment]
Echoing a comment on the original PR, this’ll buff the shit out of gangs, rev, and cult (no guns or shields), and nukies too (no guns). On LRP, interdepartmental cooperation is nonexistent and good luck profiting off a miasma chamber or something on an hour long round. MRP can survive this but it’s lowpop as always. No one liked it on /tg/ (majority I mean) and no one’ll like it here.
